explainer.md

```diff
@@ -42,43 +42,47 @@ VR provides an interesting canvas for artists looking to explore the possibiliti
 
 The basic steps any WebVR application will go through are:
 
- 1. Request a list of the available VR devices.
- 2. Checks to see if the desired device supports the presentation modes the application needs.
- 3. If so, application advertises VR functionality to the user.
- 4. User performs an action that indicates they want to enter VR mode.
- 5. Request a VR session to present VR content with.
- 6. Begin a render loop that produces graphical frames to be displayed on the VR device.
- 7. Continue producing frames until the user indicates that they wish to exit VR mode.
- 8. End the VR session.
+ 1. Request a VR device.
+ 1. If a device is available, application advertises VR functionality to the user.
+ 1. Request a VR session from the device in response to a user activation event.
```

Comment:
`user activation event` -> `user-activation event`

and can you link to https://html.spec.whatwg.org/multipage/interaction.html#activation?

Reply:
I think the link would be helpful, because this topic seems to come up a lot of `How is user intent defined in WebVR?`